services/api/src/schema.js

```diff
@@ -245,7 +245,7 @@ const typeDefs = gql`
     allProjects(createdAfter: String, gitUrl: String): [Project]
     allCustomers(createdAfter: String): [Customer]
     allOpenshifts: [Openshift]
-    allEnvironments(createdAfter: String): [Environment]
+    allEnvironments(createdAfter: String, envType: EnvType): [Environment]
```

Comment:
We have another filter within Projects to filter for the type of environments, but there it's just called `type`:
https://github.com/amazeeio/lagoon/blob/d7c190f56b87ad597e74d15f056268353087226f/services/api/src/schema.js#L188
therefore, let's rename this also to `type`